src/main/java/org/sert2521/powerup/util/TimeSync.kt

```diff
@@ -0,0 +1,42 @@
+package org.sert2521.powerup.util
+
+import java.net.DatagramPacket
+import java.net.DatagramSocket
+import java.net.InetAddress
+import java.util.Date
+
+object TimeSync : Thread() {
+    // Time (in secs) to wait between syncs
+    private const val WAIT_PERIOD: Long = 5
```

Comment:
Just do this in milliseconds, will make everything easier.

Reply:
Also no need to specify the type.

Reply:
Actually, we do need to specify this as `Long`, otherwise it gets inferred as an `Int`.